compiler/ballerina-lang/src/main/java/io/ballerina/projects/PackageManifest.java

```diff
@@ -33,12 +41,12 @@
 
     // Other entries hold other key/value pairs available in the Ballerina.toml file.
     // These keys are not part of the Ballerina package specification.
-    private final Map<String, Object> otherEntries;
+    private final Map<String, TopLevelNode> otherEntries;
```

Comment:
otherEntries should be a generic type and shouldn't be Toml specific. We can use Maps and List of strings to represent the data values. and keep the type as it is.